sympy/series/series.py

```diff
@@ -61,3 +62,107 @@ def series(expr, x=None, x0=0, n=6, dir="+"):
     """
     expr = sympify(expr)
     return expr.series(x, x0, n, dir)
+
+
+def inversion_series(f, x, a=0, n=3):
```

Comment:
Sphinx directive for the documentation should be added.

Reply:
@sylee957 sorry I am not following, would you be able to share an example (point to an existing piece of code)?

Reply:
The docstrig isn't automatically visible for HTML documentation unless you edit the rst files 
https://github.com/sympy/sympy/blame/master/doc/src/modules/series/series.rst

Reply:
You should add `.. autofunction:: sympy.series.series::inversion_series`